experimenter/experimenter/features/manifests/apps.yaml

```diff
@@ -3,22 +3,35 @@
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 ---
 fenix:
-  repo: \@mozilla-mobile/firefox-android
-  fml_path: fenix/app/nimbus.fml.yaml
+  slug: fenix
+  repo: "mozilla-mobile/firefox-android"
+  fml_path: "fenix/app/nimbus.fml.yaml"
+  channels: ["release", "beta", "nightly", "developer"]
```

Comment:
We can't discover these from the FMLs?

Reply:
This is temporary to get things working. We either have to add code here to read and parse the remote FML file to discover channels *or* add support to nimbus-cli to discover channels or add an --all-channels flag to single-file command.

Right now we need to specify a channel for single-file and the resulting single file yaml only has one channel listed.

Reply:
Perfect 👍